zipkin-storage/elasticsearch-jest/README.md

```diff
@@ -0,0 +1,23 @@
+# storage-elasticsearch-jest
+
+Implements a client provider for Elasticsearch that uses Elasticsearch's HTTP APIs by way of the [Jest client](https://github.com/searchbox-io/Jest) for flexibility.
+
+See the main elasticsearch module for more details.
+
+## Testing this component
+This module conditionally runs integration tests against a local Elasticsearch instance.
+
+Tests are configured to automatically access Elasticsearch started with its defaults.
+To ensure tests execute, download an Elasticsearch 2.x distribution, extract it, and run `bin/elasticsearch`. 
```

Comment:
Fairly well – they got rid of the `string` data type (I think we want `keyword`, since we're treating ES as a big `String.equals` engine and not doing "full text" search), but with a few changes I can get many of the tests to pass. The rest fail because ES goes into overdrive and pegs my CPU without returning any results within 10 seconds. The errors it does log suggest I've missed some important fields (i.e. "traceId" is still "text" and not "keyword"), but I stopped there for now.